src/app/Fake.DotNet.Cli/GlobalJson.fs

```diff
@@ -0,0 +1,60 @@
+﻿module internal GlobalJson
+
+open System.IO
+open Newtonsoft.Json.Linq
```

Comment:
I would switch to `System.Text.Json` as the next step in the next PR

Reply:
I've had that idea on the thought pile as it'd be nice to use the inbox version of System.Text,Json instead of needing Newtonsoft, though I also need to see how you go about including the STJ NuGet package in the .NET Standard 2.0 build and using the inbox version in the .NET 6.0 build

Reply:
rebased on top of the System.Text.Json change